index.d.ts

```diff
@@ -14,9 +14,9 @@ export const tools: { env: { inject: (filename?: string) => void } };
 export const createAppTester: (
   appRaw: object
 ) => <T extends any, B extends Bundle>(
-  func: (z: ZObject, bundle: B) => Promise<T>,
-  bundle?: Partial<B> // partial so we don't have to make a full bundle in tests
-) => T extends Promise<T> ? T : Promise<T>;
+    func: (z: ZObject, bundle: B) => Promise<T>,
+    bundle?: Partial<B> // partial so we don't have to make a full bundle in tests
+  ) => T extends Promise<T> ? T : Promise<T>;
```

Comment:
super minor, but it looks like the changes here are breaking rank w/ `prettier`? I tried it on my machine and these were reverted. We have a hook that _should_ handle this for you, but maybe it didn't run?

In any case, i'd run this file through `prettier` and re-commit just so we keep that consistent style.

Reply:
fixed this up, I must have had some bad prettier settings